_posts/2023-12-28-release.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Release 2023.52
+author: Priya
+date: 2023-12-28
+category: Jekyll
+layout: post
+---
+### Enhancements:
+#### BigQuery export - partitioning on date columns.
+Users can now enable partitioning on date columns when exporting to BigQuery for enhanced efficiency and cost savings.
```

Comment:
Mammoth now supports partitioning on date columns when exporting to Big Query. This helps reduce query costs for time series data while making the queries faster.